WordPress/Classes/Utility/ZendeskUtils.swift

```diff
@@ -179,31 +295,40 @@ private extension ZendeskUtils {
         return (allBlogs.map { $0.logDescription() }).joined(separator: Constants.blogSeperator)
     }
 
-    func getTags() -> [String] {
-
-        var tags = [String]()
+    static func getTags() -> [String] {
 
         let context = ContextManager.sharedInstance().mainContext
-        let accountService = AccountService.init(managedObjectContext: context)
+        let blogService = BlogService(managedObjectContext: context)
 
-        if let defaultAccount = accountService.defaultWordPressComAccount() {
-
-            tags = defaultAccount.blogs.filter {
-                $0.planTitle != nil
-                }.map {
-                    $0.planTitle!
-                }.unique
+        // If there are no sites, then the user has an empty WP account.
+        guard let allBlogs = blogService.blogsForAllAccounts() as? [Blog], allBlogs.count > 0 else {
+            return [Constants.wpComTag]
+        }
 
-            tags.append(Constants.wpComTag)
+        // Get all unique site plans
+        var tags = allBlogs.filter {
+            $0.planTitle != nil
+            }.map {
+                $0.planTitle!
+            }.unique
```

Comment:
(Unless i'm missing something!!) this should be equivalent to this snippet:

```
var tags = allBlogs.flatMap { $0.planTitle }.unique
```

`flatMap` should remove the `nil` values automatically for you. Super cool!